cmd/podman/machine/os/apply.go

```diff
@@ -0,0 +1,38 @@
+//go:build (amd64 || arm64) && experimental
+// +build amd64 arm64
+// +build experimental
+
+package machineos
+
+import (
+	"fmt"
+
+	"github.com/containers/podman/v4/cmd/podman/machine"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/validate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	applyCmd = &cobra.Command{
+		Use:               "apply",
+		Short:             "Apply OS image to existing VM",
```

Comment:
s/OS image/OCI image

wdyt ?

Reply:
I think OCI image is technically more correct but I'm concerned about it being confusing to someone who isn't familiar with OSTree layering. But we also have the long desc that should clear it up? Not sure on the right answer, but definitely open to re-wording.  :)

Reply:
This is going to cause havoc with man page checker once experimental is turned on.  Is this just so you and @baude can work together on the os commands?

Reply:
I think by the time we have manpages we'll have removed the experimental build tag and have it just as a regular command. I think we're just using the experimental tag to hide it from other builds ATM if I'm not mistaken.